### PR TITLE
[E2E tests] Changes needed to reuse the tests

### PR DESCRIPTION
**Pull request by Katka92**
### What does this PR do?
Changes to the code that make it reusable by e.g. Rh-Che. 
It adds:
- USERNAME and PASSWORD that would be required in MultiUser approach (and will be used on Rh-Che side too)
- index.ts to export all classes that have to be imported in downstream project
- changing ``` name ``` of the project in package.json to be the same as module name

The code was tested as SingleUser version of che on minishift
Che version: 7.0.0-beta-5.0

### What issues does this PR fix or reference?
Part of https://github.com/redhat-developer/che-functional-tests/issues/515

### CODE REVIEW

**Review by rhopp — APPROVED** [review]
LGTM, but please wait for yet someone else's approval ;-)

**Comment by rhopp** [comment]
ci-test

**Comment by musienko-maxim** [comment]
Can one of the admins verify this PR?

**Comment by rhopp** [comment]
@Katka92 Please add signoff to the commits.

**Comment by Katka92** [comment]
ci-test

**Comment by Katka92** [comment]
ci-test

**Comment by Katka92** [comment]
ci-test

**Review by ibuziuk — APPROVED** [review]

**Comment by Katka92** [comment]
ci-test

File: e2e/TestConstants.ts
Comment (ppitonak) [inline e2e/TestConstants.ts:90]: Project is no longer named `typescript-selenium` so these variables could be renamed as well (probably in a separate PR)

**Review by ppitonak — APPROVED** [review]

File: e2e/index.ts
Comment (Ohrimenko1988) [inline e2e/index.ts]: Please add empty row

**Review by Ohrimenko1988 — APPROVED** [review]

File: e2e/index.ts
Comment (Ohrimenko1988) [reply e2e/index.ts]: And clarify please, what a purpose of adding this exports?

Comment (Katka92) [reply e2e/index.ts]: These exports are needed for downstream tests to be able to reuse logic that is already implemented in upstream. You can see an example e.g. here https://github.com/redhat-developer/rh-che/pull/1438/files#diff-81af6d743c37c29cab3f9be6413a2365

**Pull request merged**
